integration_tests/macros/assert_equal_values.sql

```diff
@@ -0,0 +1,32 @@
+{% macro assert_equal_values(actual_object, expected_object) %}
+{% if not execute %}
+
+    {# pass #}
+
+{% elif actual_object != expected_object %}
+
+    {% set msg %}
+    Expected did not match actual
+
+    -----------
+    Actual:
+    -----------
+    --->{{ actual_object }}<---
+
+    -----------
+    Expected:
+    -----------
+    --->{{ expected_object }}<---
+
+    {% endset %}
+
+    {{ log(msg, info=True) }}
+
+    select 'fail'
+
+{% else %}
+
+    select 'ok' limit 0
```

Comment:
This should be [`{{ limit_zero() }}`](https://github.com/dbt-labs/dbt-utils/blob/main/integration_tests/macros/limit_zero.sql) - not all databases support limit clauses (notably [T-SQL](https://github.com/dbt-msft/tsql-utils/blob/main/integration_tests/dbt_utils/macros/limit_zero.sql))

Reply:
Done!
@joellabes -- this is no longer required, as I switched to using the seed/model method. Should I remove this file then?

Reply:
I think that makes sense - the Core team are working on a wider testing suite that I hope will also cover packages - if that wasn't in the pipeline then I'd suggest we kept this around because "we’d need it one day" but I think we'll have a more long term solution soon.